src/dos/program_imgmount.cpp

```diff
@@ -230,7 +233,7 @@ void IMGMOUNT::Run(void) {
 
     if (fstype=="fat") {
         if (imgsizedetect) {
-            FILE * diskfile = fopen_wrap(temp_line.c_str(), "rb+");
+            FILE * diskfile = fopen_wrap_ro_fallback(temp_line.c_str(), roflag);
```

Comment:
Should be able to drop `.c_str()`

Reply:
Right, fopen_wrap_ro_fallback takes std::string as an argument rather than char array. Done.